test/interval_tests/ad.jl

```diff
@@ -0,0 +1,14 @@
+using IntervalArithmetic, ForwardDiff
+using Test
+
+@testset "AD" begin
+    @test ForwardDiff.derivative(abs, -2.0 .. 2.0) == -1.0 .. 1.0
```

Comment:
```suggestion
using Test

@testset "AD" begin
    for F in (Interval, DecoratedInterval)
        @test ForwardDiff.derivative(abs, F(-2.0 .. 2.0)) == F(-1.0 .. 1.0)
```
This adaptation to all tests  would allow to test both interval and decorated intervals in one go.